include/rapidjson/prettywriter.h

```diff
@@ -83,7 +87,10 @@ class PrettyWriter : public Writer<OutputStream, SourceEncoding, TargetEncoding,
     /*! \param options Formatting options.
     */
     PrettyWriter& SetFormatOptions(PrettyFormatOptions options) {
-        formatOptions_ = options;
+        if ( options == kFormatDefault)
```

Comment:
I think this is not obvious for user. The original one looks more simple for user.